src/writeData/subscriptions/components/BrokerForm.tsx

```diff
@@ -0,0 +1,399 @@
+// Libraries
+import React, {FC, useEffect, useState, useRef} from 'react'
+import {useHistory} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+
+// Components
+import {
+  Input,
+  Button,
+  Grid,
+  Form,
+  Overlay,
+  Columns,
+  InputType,
+  ButtonType,
+  Dropdown,
+  ComponentColor,
+  ComponentStatus,
+  SelectGroup,
+  ButtonShape,
+  IconFont,
+  Icon,
+  Heading,
+  HeadingElement,
+  FontWeight,
+  FlexBox,
+  ComponentSize,
+  AlignItems,
+  FlexDirection,
+} from '@influxdata/clockface'
+
+// Utils
+import {getOrg} from 'src/organizations/selectors'
+import {handleValidation} from 'src/writeData/subscriptions/utils/form'
+import {convertUserInputToNumOrNaN} from 'src/shared/utils/convertUserInput'
+
+// Types
+import {SUBSCRIPTIONS, LOAD_DATA} from 'src/shared/constants/routes'
+import {Subscription} from 'src/types/subscriptions'
+
+// Styles
+import 'src/writeData/subscriptions/components/BrokerForm.scss'
+
+interface Props {
+  formContent: Subscription
+  setFormActive: (string) => void
+  updateForm: (any) => void
+  showUpgradeButton: boolean
+}
+
+const BrokerForm: FC<Props> = ({
+  formContent,
+  setFormActive,
+  updateForm,
+  showUpgradeButton,
+}) => {
+  const history = useHistory()
+  const org = useSelector(getOrg)
+  const mqttProtocol = 'MQTT'
+  const protocolList = [mqttProtocol]
+  const [protocol, setProtocol] = useState(mqttProtocol)
+  const [security, setSecurity] = useState('none')
+  // form validation component is in an err
+  // state once the value changes
+  // this fixes it for initial render
+  // later forms are stuck in an inital validation
+  // state though
+  const didMount = useRef(false)
+  useEffect(() => {
+    if (didMount.current) {
+      updateForm({...formContent, protocol: protocol.toLowerCase()})
+    }
+    didMount.current = true
+  }, [protocol])
```

Comment:
I'm having issues with the Form.Validation element. When it first loaded they would all be in an error state as the component was rendering twice. I added this did mount check in to stop it from calling the validation function before the user has interacted with the form. However once they interact with one form element the other validation elements are triggered - not sure if I'm missing something.

Reply:
Found the issue - it was a bug in clockface https://github.com/influxdata/clockface/pull/744

Reply:
awesome thank you for tracking this down, I will bump to 3.5.1 when it's released